client/src/reducers/authenticationReducer.js

```diff
@@ -43,9 +43,13 @@ export const authenticationReducer = (state = initialState, action) => {
     case LOGIN_SUCCESS:
       return {
         ...state,
+        login: {
+          isLoading: false,
+          error: null
+        },
         user: {
           authenticated: true,
-          username: 'username', //update the be login endpoint to return username
+          username: action.payload.username, //update the be login endpoint to return username
```

Comment:
Seems like this is a `TODO` to make a change on the backend? If this handled, maybe delete the comment.